.sanity-ansible-ignore-2.9.txt

```diff
@@ -0,0 +1,47 @@
+tests/network/ensure_provider_tests.py compile-2.7!skip
```

Comment:
It would be nice to sort this list to ensure there are no duplicates or contradictions.

Reply:
done